n3fit/src/n3fit/model_gen.py

```diff
@@ -86,7 +86,7 @@ def __call__(self, pdf_layer, mask=None):
 
 
 def observable_generator(
-    spec_dict, positivity_initial=1.0, integrability=False
+    spec_dict, positivity_initial=1.0, integrability=False, name_suffix=''
```

Comment:
I'm guessing the `name_suffix` is so that different observable are treated differently so one can have different trvl masks. Please add to the docstr.